sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaProperties.java

```diff
@@ -112,4 +118,16 @@ public static KafkaProperties forProducer(KafkaConfiguration configuration) {
 
     return props;
   }
+
+  private static Class<?> getLoginModule(String saslMechanism) {
+    switch (saslMechanism) {
```

Comment:
Might be nicer to add `toUpperCase` here.

Reply:
Yes. That might prevent potential frustrations.